### PR TITLE
Scroll vertical-horizontal visibility seperation

### LINKED ISSUE #9208: Horizontal scrollbar should not appear when scrolling vertically
## Context
Horizontal scroll bar appears even if we are not scrolling horizontally.

https://github.com/user-attachments/assets/3d6dba56-2072-4eb5-b4fc-5505ed6b5eed

### PR DESCRIPTION
**Pull request by ehconitin**
fixes #9208

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/utilities/scroll/components/ScrollWrapper.tsx [hunk 1]
```diff
@@ -75,22 +75,76 @@ export const ScrollWrapper = ({
...
+          if (scrollbar !== null) {
+            scrollbar.track.style.visibility = 'hidden';
+          }
+        });
+
+        // Show appropriate scrollbar based on scroll direction
+        const isHorizontalScroll =
+          target.scrollLeft !== Number(target.dataset.lastScrollLeft || '0');
+        const isVerticalScroll =
+          target.scrollTop !== Number(target.dataset.lastScrollTop || '0');
```
[the remark this answers is not in the document]
Comment (ijreilly) [reply packages/twenty-front/src/modules/ui/utilities/scroll/components/ScrollWrapper.tsx:106]: This seems like a good idea, wdyt?

[the remark this answers is not in the document]
Comment (ijreilly) [reply packages/twenty-front/src/modules/ui/utilities/scroll/components/ScrollWrapper.tsx:106]: although I did not notice anything annoying at testing

**Review by ijreilly — APPROVED** [review]
mostly tested it, works fine + does not cause new re renders from what i observed

File: packages/twenty-front/src/modules/ui/utilities/scroll/components/ScrollWrapper.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ehconitin) [reply packages/twenty-front/src/modules/ui/utilities/scroll/components/ScrollWrapper.tsx:106]: 🤔 @lucasbordeau wdyt?

[the remark this answers is not in the document]
Comment (lucasbordeau) [reply packages/twenty-front/src/modules/ui/utilities/scroll/components/ScrollWrapper.tsx:106]: If there's no noticeable performance issue I think that we should ignore this comment.

**Review by lucasbordeau — APPROVED** [review]

**Pull request merged**